homeassistant/components/sharkiq/vacuum.py

```diff
@@ -101,7 +119,9 @@ def send_command(
         **kwargs: Any,
     ) -> None:
         """Send a command to the vacuum. Not yet implemented."""
-        raise NotImplementedError()
+        raise NotImplementedError(
+            "Service `send_command` is currently not compatible with SharkIQ."
+        )
```

Comment:
so.. not 100% sure what you meant, but I figured this whole function could just be deleted since it's just overiding a function from the base vacuum service that will already throw an unimplemented error.  If this isn't wan't you meant let me know